packages/catalog-process/test/catalogService.integration.test.ts

```diff
@@ -4017,61 +4017,67 @@ describe("database test", async () => {
     });
 
     describe("update Document", () => {
-      it("should write on event-store for the update of a document", async () => {
-        const descriptor: Descriptor = {
-          ...mockDescriptor,
-          state: descriptorState.draft,
-          docs: [mockDocument],
-        };
-        const eservice: EService = {
-          ...mockEService,
-          descriptors: [descriptor],
-        };
-        await addOneEService(eservice, postgresDB, eservices);
-        await catalogService.updateDocument(
-          eservice.id,
-          descriptor.id,
-          mockDocument.id,
-          { prettyName: "updated prettyName" },
-          {
-            authData: getMockAuthData(eservice.producerId),
-            correlationId: "",
-            serviceName: "",
-            logger: genericLogger,
-          }
-        );
-        const writtenEvent = await readLastEserviceEvent(
-          eservice.id,
-          postgresDB
-        );
-        const expectedEservice = toEServiceV2({
-          ...eservice,
-          descriptors: [
+      it.each(
+        Object.values(descriptorState).filter(
+          (state) => state !== descriptorState.archived
+        )
+      )(
+        "should write on event-store for the update of a document for draft descriptor",
```

Comment:
Could you edit the name of the test? It could be: `"[...] update of a document in a non-archived descriptor"`.